core/shared/src/main/scala-3/me/shadaj/scalapy/py/Module.scala

```diff
@@ -0,0 +1,20 @@
+package me.shadaj.scalapy.py
+
+import scala.language.dynamics
+import scala.reflect.ClassTag
+
+import me.shadaj.scalapy.readwrite.FacadeReader
+import me.shadaj.scalapy.interpreter.CPythonInterpreter
+
+@native class Module extends Dynamic
+
+object Module {
```

Comment:
Companion object can be shared with Scala 2

Reply:
bump? we can create a trait with the `apply` methods that's mixed into the companion object in Scala 2/3